RELEASE.rst

```diff
@@ -47,8 +47,10 @@ How to make a new release of ``networkx``
 - Publish on PyPi::
 
    git clean -fxd
-   python setup.py sdist --formats=zip
-   twine upload -s dist/networkx*.zip
+   pip install -U pip setuptools wheel
+   pip intsall -U twine
```

Comment:
This has a typo `intsall` -> `install`, maybe that's why it didn't work and was reverted in commit https://github.com/networkx/networkx/commit/61f708abb17d45fc8dd1a04c26a454932fc74274 ?